tests/Tus/ServerTest.php

```diff
@@ -714,28 +714,6 @@ public function it_handles_head_request_for_final_upload()
         $this->assertEquals('no-store, private', $response->headers->get('cache-control'));
     }
 
-    /**
-     * @test
-     *
-     * @covers ::handlePost
-     */
-    public function it_returns_400_for_invalid_upload()
-    {
-        $this->tusServerMock
-            ->getRequest()
-            ->getRequest()
-            ->server
-            ->add([
-                'REQUEST_METHOD' => 'POST',
-                'REQUEST_URI' => '/files',
-            ]);
-
-        $response = $this->tusServerMock->handlePost();
-
-        $this->assertEquals(400, $response->getStatusCode());
-        $this->assertEmpty($response->getContent());
-    }
-
```

Comment:
Why was this removed ?

Reply:
400 was returned if the filename was missing. With this PR, sending filename is optional because there's a fallback, this test is not needed anymore.

https://github.com/ankitpokhrel/tus-php/blob/3878a4933d9e62d4e9a543e88d2b76fd4787d0e1/src/Tus/Server.php#L347-L349

Reply:
But if we remove this test then our test coverage will drop right? I think it's better to keep test intact even if fallback covers it. Or better remove this code as well. @ankitpokhrel What do you think?

Reply:
@alucic we can remove `if (empty($fileName)) { ` check since that piece of code is not reachable anymore and it is better to add a test to make sure we are getting auto generated filename if it is not provided in the meta.

Reply:
Isn't that covered with `it_extracts_file_name_has_fallback`? Am I missing something?

Reply:
I would also add one test it in `ServerTest.php` by not passing `Upload-Metadata` header, it should take the `UUID` in that case.

Reply:
Oh, I just reread the comments and figured why there was a confusion, I didn't remove that it if statement that returns 400.

I'll update the PR to use uploadKey instead generating a new UUID for the filename